packages/cli-repl/test/e2e-tls.spec.ts

```diff
@@ -221,14 +215,12 @@ describe('e2e TLS', () => {
         shell.assertContainsOutput(`user: '${certUser}'`);
       });
 
-      // TODO: enable when NODE-2977 is solved
-      // eslint-disable-next-line mocha/no-skipped-tests
-      it.skip('works with valid cert (connection string)', async() => {
+      it('works with valid cert (connection string)', async() => {
         const shell = TestShell.start({
           args: [
             `${await server.connectionString()}?serverSelectionTimeoutMS=1500`
             + '&authMechanism=MONGODB-X509'
-            + `&tls=true&tlsCAFile=${encodeURIComponent(CA_CERT)}&tlsCertificateKeyFile=${encodeURIComponent(CLIENT_CERT)}`
+            + `&tls=true&tlsCAFile=${encodeURIComponent(CA_CERT)}&tlsCertificateKeyFile=${encodeURIComponent(CLIENT_CERT)}&tlsCertificateFile=${encodeURIComponent(CLIENT_CERT)}`
```

Comment:
Waiting for the driver team to get back to us about this on slack

Reply:
Also looks like this is something we need to deal with for now. Do we want to do some URL rewriting for cases like this? Do we want to just ignore this and accept the driver change as it is?

Reply:
Okay, we’re doing URL rewriting now. The test change here is undone now :)